tsup.config.ts

```diff
@@ -3,14 +3,31 @@ import { promises as fsPromises } from "fs";
 import path from "path";
 import postcss from "postcss";
 import postcssModules from "postcss-modules";
+import { glob } from "glob";
 
 export default defineConfig({
-  entry: ["src/index.tsx"],
+  entry: glob.sync("src/**/*.{ts,tsx}", {
+    cwd: process.cwd(),
+    ignore: [
+      "**/node_modules/**",
+      "**/dist/**",
+      "**/cjs/**",
+      "**/esm/**",
```

Comment:
```suggestion
```

I think these here are overkill, left over from my WIP branch that this stuff building out into different folders with a bunch more code in the `scripts` dir.  none of that was necessary for this PR